src/DatePicker/DatePicker.js

```diff
@@ -221,54 +112,18 @@ class DatePicker extends Component {
 
     updateDate = (date) => {
         if (this.props.enableRangeSelection) {
-            if (date.length === 2) {
-                let firstDateMonth = date[0].getMonth() + 1;
-                let firstDateDay = date[0].getDate();
-                let firstDateYear = date[0].getFullYear();
-
-                let secondDateMonth = date[1].getMonth() + 1;
-                let secondDateDay = date[1].getDate();
-                let secondDateYear = date[1].getFullYear();
-
-                let formatDate =
-                    firstDateMonth +
-                    '/' +
-                    firstDateDay +
-                    '/' +
-                    firstDateYear +
-                    '-' +
-                    secondDateMonth +
-                    '/' +
-                    secondDateDay +
-                    '/' +
-                    secondDateYear;
-                this.setState({
-                    arrSelectedDates: date,
-                    formattedDate: formatDate
-                });
-            } else {
-                let firstDateMonth = date[0].getMonth() + 1;
-                let firstDateDay = date[0].getDate();
-                let firstDateYear = date[0].getFullYear();
-
-                let formatDate =
-                    firstDateMonth + '/' + firstDateDay + '/' + firstDateYear;
-
-                this.setState({
-                    arrSelectedDates: date,
-                    formattedDate: formatDate
-                });
+            let formatDate = date[0].format('MM/DD/YYYY');
+            if (!!date[1]) {
+                formatDate += '-' + date[1].format('MM/DD/YYYY');
```

Comment:
Instead of hardcoding month/day/year could we use `L` and set the locale so it's always localized?